src/main/java/com/quiz/model/User.java

```diff
@@ -3,73 +3,48 @@
 import java.util.List;
 
 public class User {
-    private Long id;
-    private String fullName;
-    private String userName;
+    private int id;
+    private String fullname;
```

Comment:
there is a standard in naming conversion in java called camel case

`userName`
`fullName`